contributing/language-translations.md

```diff
@@ -0,0 +1,38 @@
+(translations)=
+# Language Translations
+
+## Requirements
+- GitHub account
+- Knowledge of English
+- Knowledge of Spanish or Portuguese
+
+## About
+We are using [Transifex](https://www.transifex.com/) to manage the language translations, which provides a translation infrastructure with Sphinx. Transifex is *a global content repository with a REST API, SDKs, automation, and more so you can centrally manage your source content and translations and build amazing multilingual digital experiences for your users.*
+
+## Workflow
+
+1.  Create a [Transifex account](https://www.transifex.com/signin/).  Note: You can use your GitHub account to authenticate and log in to Transifex.
+
+2.  Navigate to our project website:  [pymc/data-umbrella-sprints-website](https://www.transifex.com/pymc/data-umbrella-sprints-website/)
+
+3. Click on "Editor" in the top menu which will bring you to a list of pages of the website to translate:  https://www.transifex.com/pymc/data-umbrella-sprints-website/translate/
+
+4. In the top menu, choose "Select a language". Current options are:
+    - Portuguese (pt)
+    - Spanish (es)
+
+5. Pages to translate is a sub-menu under "Editor."  Currently, these pages are being prioritized for translation:  [priority pages](https://github.com/pymc-devs/pymc-data-umbrella/issues/99)
+
+6. You can translate sub-sections and select "Save translation."  
+
+:::{note}
+There is GitHub Actions that is configured on this repository, which turns a translation into a pull request which is submitted to the repository. [PR #166](https://github.com/pymc-devs/pymc-data-umbrella/pull/166/files) is an example pull request.
+:::
+
+## Translating content: volunteers needed
+
+If you would like to volunteer for translations, please select a page on [Issue #99](https://github.com/pymc-devs/pymc-data-umbrella/issues/99) on GitHub and indicate which page you will translate. A comment can be added to the issue.  This will let other contributors know so that work is not duplicated.
```

Comment:
I would not have them comment. Transifex is updated live so there isn't really much room for working on the same thing, but it complicates the workflow and adds a bit more overhead on our side too. I would start having them check the pages on the issue and then going to transifex straight away. If it has translations already don't repeat them, if it doesn't work on transifex